project/en-US/compensations/compensations.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE chapter PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN" "http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd" [
+        <!ENTITY % BOOK_ENTITIES SYSTEM "../project_documentation.ent">
+        %BOOK_ENTITIES;
+        ]>
+<chapter>
+    <title>Compensations Framework</title>
+    <section>
+        <title>Outstanding Important Issues</title>
+        <itemizedlist>
+            <listitem>
+                <para>[JBTM-1107] Recovery Support in Compensation API</para>
```

Comment:
I understand, but it is still important to make for the people to know that recovery is not available at the moment. We can take it out once recovery is implemented.